Cargo.toml

```
@@ -105,6 +105,7 @@ url = "2.4"
 bytes = "1.0"
 serde = "1.0"
 serde_urlencoded = "0.7.1"
+tower = { version = "0.5", default-features = false, features = ["timeout", "util"] }
```

https://github.com/seanmonstar/reqwest/actions/runs/12300938632/job/34330594262?pr=2496#step:4:404

```
error[E0432]: unresolved import `tower::util::BoxCloneSyncService`
  --> src/connect.rs:12:48
   |
12 |     layer::util::Stack, timeout::TimeoutLayer, util::BoxCloneSyncService, Layer, ServiceBuilder,
   |                                                ^^^^^^-------------------
   |                                                |     |
   |                                                |     help: a similar name exists in the module: `BoxCloneService`
   |                                                no `BoxCloneSyncService` in `util`
```

0.5.2 is needed because we use `tower::util::BoxCloneSyncService`.

https://diff.rs/tower/0.5.1/0.5.2/src%2Futil%2Fboxed_clone_sync.rs


```suggestion
tower = { version = "0.5.2", default-features = false, features = ["timeout", "util"] }
```

---

Thanks! Will update that.